Tests/KlaviyoCoreTests/__Snapshots__/KlaviyoAPITests/testSuccessfulResponseWithEvent.1.txt

```diff
@@ -13,8 +13,7 @@
       ▿ (2 elements)
         - key: "X-Klaviyo-Attempt-Count"
         - value: "0/50"
-  ▿ httpBody: Optional<Data>
-    - some: 0 bytes
+  - httpBody: Optional<Data>.none
```

Comment:
just making sure this is expected.

Reply:
yes, because of [this](https://github.com/klaviyo/klaviyo-swift-sdk/pull/240/files#diff-6e841d12ea7c6ab9ce7088a7d3cc2b511c8436e34fce587c89f0b3789cb38afbR45) change